### PR TITLE
Phone pe

### PR DESCRIPTION
**Pull request by Damini2004**
Closes: #314 

- **Title** :  Phonepe Clone
- **Name:** : Damini Chachane
- **Idenitfy yourself:** : GSSOC

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Front-end-Projects/Basic/PhonePe/ScreenShot.webp
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/PhonePe/ScreenShot.webp:1]: follow the naming conventions and don't forget to update the your project README.md file

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by GAVINESHWAR** [comment]
@Damini2004 I ma getting error while using nom start

PS C:\Users\Avineshwar G\Documents\internship\opensource\WebMasterLog> npm start
npm ERR! Missing script: "start"
npm ERR!
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: C:\Users\Avineshwar G\AppData\Local\npm-cache\_logs\2024-06-21T05_25_24_324Z-debug-0.log

Can you say how you open the project

**Comment by Damini2004** [comment]
@GAVINESHWAR  The project is designed with the help of HTML, CSS not in react, node js so no need to run command of "npm start" in order to run just run the index.html file

**Comment by Damini2004** [comment]
@Avdhesh-Varshney Kindly review the PR

**Comment by Damini2004** [comment]
@Avdhesh-Varshney Can you increase the level.

File: Front-end-Projects/Basic/PhonePe/ScreenShot.webp
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/PhonePe/ScreenShot.webp:1]: you have updated the README.md but not updating the name of the file?

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Pull request closed without merging**